# Spatial omics

Reviewed: No
Type: Slides

# **Spatial Transcriptomics**

A range of methods for **quantifying RNAs** in **specific locations** on histological sections *“retaining information on spatial context”.* Important criteria:

- *How many RNAs?* → throughput
- *At which resolution?* → the resolution is the unit of space that can be discriminated in the image. We cannot have a grid perfectly adapting to single cells, so we will observe pieces of different cells rather than single cells.

If we look at the evolution of main ST platforms, we observe a correspondence with single cell techniques with a 4-5 years gap. 

The main ST approaches can be divided into:

1. *imaging based methods*: read out by microscopy e.g. multiplexed FISH
    - Pros: high resolution, depending on the microscope
    - Cons: low coverage or low number of RNAs, need to design target probes (not possible to have more than 10 fluorophores)
2. *sequencing based methods*: spatial barcoding, read out by sequencing - more similar to single cell approaches. The cells are placed on a plate covered with adapters (UMI + polyA sequence capturing mRNAs different according to the location).
    - Pros: high coverage
    - Cons: low resolution

# Sequencing-based methods

In **NGS barcoding techniques** spatial locations of transcripts obtained by capturing the transcripts from tissue sections on in situ arrays

- How many RNAs? several thousands/ unbiased capture (but mainly polyA)
- At which resolution? Not cellular (pixels with 5-50 μM resolution)

## Spatial transcriptomics (2016)

The adult mouse olfactory bulb brain is a region with clear histological landmarks and gene expression reference data. Histological sections positioned on **arrays with printed spot barcodes** (spatial barcode, UMI, polyT to capture mRNAs) *(~1k spots, spot diameter of 100 μm, center-to-center distance of 200 μm)* were analyzed by  Stahl et al,…

## 10x Visium platform for spatial gene expression (2019)

![Screenshot.png](Spatial%20omics%20ef9a63e8a3e44636b6e607287c5920c3/Screenshot.png)

## Slide-seq (2019)

![Screenshot.png](Spatial%20omics%20ef9a63e8a3e44636b6e607287c5920c3/Screenshot%201.png)

## DBiT-seq (2020)

DBiT(Deterministic Barcoding in Tissue) is an array generated by microfluidic channels: 2 orthogonal barcodes ligated, spots identified with a unique pairwise combination (A for rows, B for columns).

Channel width: 50, 25, or 10 μm. 50x50 grid (2500 spots). It allows for parallel spatial transcriptomic profiling and antibody-based protein barcoding.

![Liu et al., Cell 2020](Spatial%20omics%20ef9a63e8a3e44636b6e607287c5920c3/Screenshot%202.png)

Liu et al., Cell 2020

## Seq-scope (2021)

resolution comparable to an optical microscope (0.5–0.8 μm)
use of a repurposed Illumina sequencing platform (MiSeq flow-cell in the publication)

Application to mice liver sections
****

![*Cho et al., Cell 2021*
](Spatial%20omics%20ef9a63e8a3e44636b6e607287c5920c3/Screenshot%203.png)

*Cho et al., Cell 2021*

## ****SiT (2022)****

Spatial Isoform Transcriptomics combines Visium platform with long read sequencing (Nanopore). It allows the identification of isoforms (splicing variants).

![*Lebrigand et al bioRxiv 2022*
](Spatial%20omics%20ef9a63e8a3e44636b6e607287c5920c3/Screenshot%204.png)

*Lebrigand et al bioRxiv 2022*

# Imaging based methods

## In-Situ Hybridization

ISH (In Situ Hybridization) is based on nucleic acid probes synthesized, **labeled**, purified, and annealed with the specific target (by complementarity). The technique can be applied to DNA and RNA with nucleic acid probes (20-50long). 

### Radioactive ISH (1967)

ISH was first used to study the formation and detection of RNA-DNA hybrid molecules in cytological preparations (*Gall and Pardue, PNAS,1969*).  ******Cells are treated in order to denature DNA and incubated with RNA to detect hybrids by **autoradiography**. Probes are able to catch radioactivity signal and are identified where the ribosomes are being produced.

### FISH (1997)

- **Direct FISH detection:** fluorescent labels attached to the probe which will hybridize to a target DNA/RNA
- **Indirect FISH detection:** biotin is attached to the probe. Streptavidin linked to a fluorescent tag binds biotin with high specificity.

Fluorescence microscopy can be used to find out where the fluorescent probe is bound.

### **Whole mount ISH (1989)**

Whole mount in situ hybridization determines the RNA expression pattern of a gene in the context of a whole embryo or embryo piece/organ (also 3D). First experiments were carried out already knowing the pattern of expression of certain genes. They selected:

1. hunchback
2. krippel
3. knirps
4. fushi tarazu

They could also look at how different was the localization between mRNA and proteins produced, with antibodies.

### RNA **smFISH (2002)**

The release of the first reference genomes, allowed the computational design of probes. Multiple probes can be used for a single transcript for increased specificity and single molecule resolution. Each probe is labeled with different labels. The combination of "colors" forming a pseudocolor is assigned to a specific transcript.

For smFISH, they used 4 different fluorophores for the identification of 10 genes. smFISH has a problem: the number of combinations (pseudogenes) is limited by the number of probes. This was solved by the next technique.

![*Levsky et al, Single-Cell Gene Expression Profiling, Science 2002*
](Spatial%20omics%20ef9a63e8a3e44636b6e607287c5920c3/Screenshot%205.png)

*Levsky et al, Single-Cell Gene Expression Profiling, Science 2002*

### **MERFISH (2015)**

Multiplexed Error-Robust FISH (MERFISH) is based on multiple rounds of hybridization and detection of signal, with consequent removal of probe for each round. The barcode is formed by checking at each round if the signal of the probe you are expecting is present on an RNA molecule (N-bit binary code: if yes, bit signal 1, else 0). They use an indirect method with two types of probes, one type that binds to RNA and one that binds to the fluorophore. 

![*Chen et al, Science 2015*](Spatial%20omics%20ef9a63e8a3e44636b6e607287c5920c3/Screenshot%206.png)

*Chen et al, Science 2015*

Encoding-readout labeling strategy allows genome-scale imaging with shorter experimental duration hybridization of FISH probes to exogenously introduced readout sequences is much faster than hybridization directly to cellular RNAs. The binary coding scheme (1 color) distinguishes 2N genes with N rounds of hybridization. When C colors are used, 2NC genes can be distinguished with N hybridization rounds.

Analysis example: simultaneous measurement of 140 RNA species in a single cell (IMR90, fibroblast). MERFISH with a 16-bit MHD4 code (modified humming distance 4 code) at least four bits must be read incorrectly to change one valid codeword into another). Error detection & correction.

### **RNA seqFISH+ (2019)**

80 rounds of hybridization with 3 color imaging (identified 47k mRNA). Tried in mouse brain slices and identified the position of transcripts with respect to all the others, in different types of cells. Is it also possible to then build the architecture of the tissue, in terms of what kind of cells appear close to others.

![Screenshot.png](Spatial%20omics%20ef9a63e8a3e44636b6e607287c5920c3/Screenshot%207.png)

### MERSCOPE platform (2021)

Commercial platform using MERFISH technology. It is possible to study spatial features at different resolution levels:

- whole section (9x7 mm) e.g. tissue organization
- wide field of view (200x200 micron) e.g. cell interaction or function
- sub-cellular (12x12 micron, equivalent to < 100 nm) e.g. L2 or 3 IT Glutamatergic neuron

Vizgen MERSCOPE automatically segments the cells in the MERFISH measurement using a segmentation method called Baysor, that optimizes cell boundaries considering joint likelihood of transcriptional composition and cell morphology (size and shape of the cell). The approach can take into account **nuclear (DAPI) or cytoplasm (poly(A)) staining**, however, can also perform segmentation based on the detected molecules alone. By tuning parameters, a better visualization can be obtained e.g. alpha for transparency, palette, etc. Using the positional information of each cell, we compute spatial niches (region of tissue). A relevant aspect into account is the doublet rate i.e. mixture of cell types in the same pixel.

### **10x Xenium platform (2023)**

Probes are designed in a slightly different way: they have two complementary arms that, if close to each other, can undergo a process of **ligation** (this allows to rule out the fact that the probe might bind to two different mRNA molecules).

### AKOYA platform (2021)

The CODEX platform developed by Akoya Biosciences is based on DNA barcoded antibodies, which allow to perform multiplexed imaging. Antibodies are specific for one protein, and the addition of fluorescent probes complementary to the DNA barcode (antibody specific) allow the visible rendering of the antibody. After signal detection, chemical stripping of fluorescent oligonucleotides occurs. The difference with previous techniques is that we capture proteins instead of transcripts.

## Summary

**Improvements:**

- Throughput (up to 1000s different RNAs)
- High resolution (subcellular)
- No polyA restriction (but sequence and splicing knowledge necessary to design complementary probes)

**Challenges:**

- optical resolution (microscope)
- density & structure of transcripts in cells (RNA granules, RNA-protein interactions could compromise probe hybridization)

# ****Spatial omics analysis****

## Deconvolution of low-resolution multi-cell spots

The aim is to uncover cellular heterogeneity in low resolution spots (e.g. Visium) to disentangle the spatial patterns of cell types. Most methods rely on the availability of a single cell annotated reference. 

- understand how many nuclei are present in a spot
- analyze cell types present in each spot

In *Li et al. Nature Communications. 2023* 18 different deconvolution methods are compared. The composition of the spot is represented as a pie chart.

![*Li et al. Nature Communications. 2023*](Spatial%20omics%20ef9a63e8a3e44636b6e607287c5920c3/Screenshot%208.png)

*Li et al. Nature Communications. 2023*

The best performing method is CARD, tool published in 2022 based on matrix factorization. A single cell reference or matrix of gene expression data is required (for cell marker identification) and the location of the spots from spatial  transcriptomics. The final results will report the estimated cell type proportion.

![Screenshot.png](Spatial%20omics%20ef9a63e8a3e44636b6e607287c5920c3/Screenshot%209.png)

## Cell segmentation in imaging-based methods

Analysis specific for imaging approaches. **Baysor** is applied to distinguish the boundaries of individual cells based on:

- Co-staining information
    - nuclei (for example, DAPI) - most common approach
    - cell bodies (for example, polyA)
    - cellular membranes
- Spatial expression data
    - increased spatial density of molecules within cell somas
    - transcriptional composition of local molecular neighborhoods

## Inference of Cell-cell communication

Reconstruct cell-cell communication from single-cell and/or spatial omics. The idea behind this is studying how cells react to stimuli from their environment(in multicellular organisms, other cells) relevant for apoptosis and cell migration, essential in homeostasis and disease. Ligands (soluble proteins) bind to receptors on the cell surface, activating cascade affecting transcriptional regulation.

![Screenshot.png](Spatial%20omics%20ef9a63e8a3e44636b6e607287c5920c3/Screenshot%2010.png)

### CellPhoneDB

The method infers crosstalk between pairs of cell groups (source and receiver) and is mainly based on expression of **ligands** (by the source) and **receptors** (on the receiver). Information about ligand-receptor interactions extracted from prior knowledge resources e.g.  PPI-interaction, protein complexes and secreted or membrane proteins databases.

Enriched receptor–ligand interactions are identified from the average expression of a ligand (source) and a receptor (target). Assumption: expression levels match protein levels. The method can be used in multiple conditions to study how communication changes upon stimuli or in disease settings. 

## Integration between single cell and spatial omics

In order to perform a comprehensive study, it is possible to combine multimodal spatial omics e.g. DbitSEQ to combine epigenome and transcriptome.

## Spatal omics reference atlases

- BICCN: diverse cell types in human, mouse and non-human primate **brain**
- HuBMAP: global atlas of human body
- Human Cell Atlas: map every cell type of the human body
- Spatial Omics DataBase (SODB):